### ISSUE TITLE
Compiler options page 'noisy'

### ISSUE THREAD

**Issue by virtuallightstudios** [microsoft/TypeScript-Website#329]
This page either needs some form of tabulation - this is just text 'noise' and is hard to scan through the options - designer should know better than this

https://www.typescriptlang.org/docs/handbook/compiler-options.html

**Comment by orta** [comment]
Hi, designer and developer on the site here - it should be better with the most recent deploy. 

It's a straight copy of the old version which had the same constraints I'm afraid, the [TSConfig Reference](https://www.typescriptlang.org/tsconfig) replaces this table

**Comment by virtuallightstudios** [comment]
I honestly think microsoft lost the ability to design usable and readable guides and User Interfaces since Windows 8 Metro HELL - to me as a user/developer both pages are a jumble of information that are hard to follow and mentally process - the newer version makes it worse by having 3 columns to process - compared to the ease of use of MDN documentation this is farcical. No wonder Microsoft consider MDN to be their source of JS docs.

Microsoft seem obsessed with making every flat and indistinguishable.

I guess this is as good as it gets from Microsoft.

**Comment by orta** [comment]
Wow, shot to the heart there @virtuallightstudios - I'm proud of this design work (it's also the first thing I did since I joined Microsoft) sorry it doesn't work for you.

**Comment by virtuallightstudios** [comment]
The comment wasn't meant as a personal attack - however it may to wise to put the design out internally and get some feedback prior to unleashing on the public.

Part of my issue is that I hate change especially as it happened suddenly within the 24 hours that I am evaluating Typescript.
